docs/PactBackendBenchmarks.md

```diff
@@ -126,3 +144,31 @@ variance introduced by outliers: 78% (severely inflated)
 ## Notes on Phase 2
 
 Would like to see if open_v2 hurts or helps pact but can't crack open `initSQLite` ...
+
+# Index problems and nested savepoints
+
+## Fixing index on versioned tables
+
+HUGE improvement: index on (rowkey, blockheight, txid), pact pragmas
+
+```
+benchmarking pact-backend/checkpointer/usertable
+time                 80.96 μs   (80.46 μs .. 81.97 μs)
```

Comment:
These numbers are awesome. Hopefully prepared statements (if possible) can bring this number down.